src/reputation/tests/test_bounties.py

```diff
@@ -1415,3 +1415,19 @@ def test_hubs_endpoint_excludes_hubs_without_bounties(self):
         self.assertEqual(response.status_code, 200)
         hub_ids = [hub["id"] for hub in response.data]
         self.assertNotIn(unused_hub.id, hub_ids)
+
+    def test_bounty_dao_fee_goes_to_community_revenue_account(self):
+        community_revenue_user, _ = User.objects.get_or_create(email="revenue1@researchhub.foundation")
```

Comment:
Since `FOUNDATION_REVENUE_EMAIL` would be ideally a configurable setting, using that variable here would be preferred.